pom.xml

```diff
@@ -70,6 +76,17 @@
 					</archive>
 				</configuration>
 			</plugin>
+			<!-- Do not enforce fail on SNAPSHOT dependencies -->
```

Comment:
That will have to go with depending on a non-SNAPSHOT release.  It's ok while hacking, not ok for a merge to master.